regolith/helpers/a_todohelper.py

```diff
@@ -0,0 +1,157 @@
+"""Helper for adding a to_do task to people.yml
+
+"""
+import datetime as dt
+import dateutil.parser as date_parser
+from dateutil.relativedelta import relativedelta
+import sys
+
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+    document_by_value,
+)
+
+TARGET_COLL = "people"
+ALLOWED_IMPORTANCE = [0, 1, 2]
+
+
+def subparser(subpi):
+    subpi.add_argument("description", help="the description of the to_do task. If the description has more than one "
+                                           "word, please enclose it in quotation marks.",
+                       default=None)
+    subpi.add_argument("due_date",
+                       help="Due date of the task. Either enter a date in format YYYY-MM-DD or an "
+                            "integer. Integer 5 means 5 days from the "
+                            "begin_date. "
+                       )
+    subpi.add_argument("-i", "--id", help="ID of the to-do list's owner. The default id is saved in user.json. ")
```

Comment:
please expand the help to be more helpful.  

Is this the id of the person who has to do the todo?  If so say so.  In other words, if you make a todo for me, would you type here jwzang or sbillinge?.   We want the default behavior to be that if it is not given it is jwzang because the expectation is that we are mostly giving ourselves todos.  So the concept of "owner" of the todo is ambiguous.  Is the owner the creator or the person it is assigned to?